inst/get_commits_workflow.R

```diff
@@ -22,9 +22,26 @@ release_logs
 test_gitstats
```

Comment:
```suggestion
git_stats
```

`test_gitstats` doesn't exist